visallo/visallo
# fix the add property list showing system properties

Testing Instructions:
- Create an entity
- Go to the inspector and add a property
- Property list should not show system properties or non-visible properties

Points of Regression: N/A

NO CHANGELOG
